### ISSUE TITLE
Visual/Sound Polish Items

### ISSUE THREAD

**Issue by gfodor** [mozilla/hubs#577]
Grab bag of polish items (not focused on UI or complex interactions) can be broken out into separate issues as needed:

- [ ] Fade-in when transitioning to immersive mode when wearing a VR headset, and play "Hubs sound"
- [ ] Fade in/out + sfx when teleporting (this may need to be an option)
- [ ] Teleporter sfx
- [ ] Ratchet turn sfx
- [ ] HUD mouse over/click/hover sfx
- [ ] Spring animation, particle system (desktop only?), and sfx on when media objects spawn from cube
- [ ] Sfx and refinement on loading cube -- cube can potentially be a random color and perhaps should pulse in scale. Color can be reflected in particle effect when media object is spawned
- [ ] Disable cursor + ray when manipulating an object, perhaps apply visual treatment to object under manipulation
- [ ] Cursor refinement: try smaller cursor size, and make cursor visible size constant regardless of distance. Experiment with change cursor color/opacity and ray color/opacity when pointing at an interactable object. (Eg, cursor should potentially be grey/less opaque when not hovering over a clickable object)
- [ ] Sfx and spring animation when letting go of object which is then "locked" in space
- [ ] Reduce head scaling for VoIP, try applying slight visual glow to head or mouth area as well if easy
- [ ] Sound effect while drawing with pen (waiting on sound design) -- might be tricky
- [ ] Sound effect when changing pen size and color
- [ ] Experiment with particle effects on pen tip while drawing
- [ ] "Broken image" should dissolve and float up and away after a few seconds, and probably not be a full interactable (on desktop, would be nice to have an error message too)
- [ ] Pause mode visual design pass: pause mode (soon to be edit mode) should have an subtle ambiance sound track mixed in and have an image space visual effect like desaturation or blur
- [ ] Nametag visual design pass: try changing the font to the mozilla font, changing it to white, and reducing the text size or making it scale based upon distance
- [ ] Smooth interpolation and perhaps a spring-overshoot and/or sfx when scaling an object. Also current scaling increments feel too small, and are in absolute terms so end up having minimal effect except within a certain scale rang. Should be bigger steps, and be an absolute scale (say 5% per scroll wheel click or something.)
